packages/devtools/client/.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Can we merge this config to the root vscode settings file

Reply:
The field is used to configure the vscode css variables plugin. Enhancing the configure and including more lookup files may cause performance issues to editor.

Reply:
I'm not sure how to handle this configuration either. Maybe we can dicuss it later?

Reply:
ok👌